.isort.cfg

```diff
@@ -1,11 +1,7 @@
 [settings]
-profile=black
-from_first=true
 import_heading_future=Future
 import_heading_stdlib=Standard
 import_heading_thirdparty=Third Party
 import_heading_firstparty=First Party
 import_heading_localfolder=Local
-known_firstparty=
-known_localfolder=fms_mo,tests
```

Comment:
I don't think we can remove this.  It was labeling fms_mo directories as first party when they should be local.

Reply:
Good catch @BrandonGroth, thanks. I am going to commit a new PR instead as have some other settings out of sync.